docs/changelog.rst

```diff
@@ -32,7 +32,9 @@ New features:
 
 Bug fixes:
 
-* Fix implicit paths OR queries (e.g. ``beet list /path/ , /other-path/``) 
+* Added Spotify 429 (too many requests) API error handling
+  :bug:`4370`
```

Comment:
Maybe we should also say the bottom line for what the user can expect here: namely, we now respect the Spotify API's rate limiting, which avoids crashing when the API reports code 429 (too many requests).

Reply:
I changed it back to `info` so that the users are not guessing. See my argument [here](https://github.com/beetbox/beets/pull/4370#issuecomment-1157634268). If you prefer `debug`, I can make the change.

Reply:
Changed logging it to `debug`.

Reply:
Thanks!!! I do think this is the right thing. The philosophy is that now, with this new strategy, 429 responses are part of the _normal operation_ of the command—it's how we're enforcing the rate limit logic. So there's nothing actionable for the user to interpret when these errors occur.